test/test_approximate_string_matching.py

```diff
@@ -2,9 +2,12 @@
 import os
 import unittest
 
+from random import randrange
 import regex
 
-from approximate_string_matching import dont_care, matching_with_dont_cares
+from approximate_string_matching import dont_care, matching_with_dont_cares \
```

Comment:
You forgot a comma at the end of the line.

Reply:
Added.